src/Arcus.Tests/MacAddressTests.cs

```diff
@@ -11,822 +11,822 @@
-        [MemberData(nameof(Comparison_MacAddress_MacAddress_Values))]
-        public void GetHashCode_Test(int expected,
-                                     MacAddress left,
-                                     MacAddress right)
-        {
-            // Arrange
-            if (left is null
-                || right is null)
-            {
-                return; // skip null values and bail
-            }
-
-            // Act
-            var leftHashCode = left.GetHashCode();
-            var rightHashCode = right.GetHashCode();
-
-            // Assert
-            Assert.Equal(expected == 0, leftHashCode == rightHashCode);
-        }
-
-        #endregion end: GetHashCode
-
-        #region GetOuiBytes
-
-        [Fact]
-        public void GetOuiBytes_Test()
-        {
-            // Arrange
-            var bytes = new byte[] {0x01, 0x23, 0x45, 0x67, 0x89, 0xAB};
-
-            var macAddress = new MacAddress(bytes);
-
-            // Act
-            var ouiBytes = macAddress.GetOuiBytes();
-
-            // Assert
-            Assert.Equal(0,
-                         ByteArrayUtils.CompareUnsignedBigEndian(bytes.Take(3)
-                                                                      .ToArray(),
-                                                                 ouiBytes));
-        }
-
-        #endregion end: GetOuiBytes
-
-        #region IsGloballyUnique
-
-        [Theory]
-        [InlineData(true, new byte[] {0b0000_0000, 0x00, 0x00, 0x00, 0x00, 0x00})]
-        [InlineData(false, new byte[] {0b0000_0010, 0x00, 0x00, 0x00, 0x00, 0x00})]
-        [InlineData(true, new byte[] {0b1111_1101, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF})]
-        [InlineData(false, new byte[] {0b1111_1111, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF})]
-        public void IsGloballyUnique_Test(bool expected,
-                                          byte[] bytes)
-        {
-            // Arrange
-            var macAddress = new MacAddress(bytes);
-
-            // Act
-            var isGloballyUnique = macAddress.IsGloballyUnique;
-
-            // Assert
-            Assert.Equal(expected, isGloballyUnique);
-        }
-
-        #endregion end: IsGloballyUnique
-
-        #region IsLocallyAdministered
-
-        [Theory]
-        [InlineData(false, new byte[] {0b0000_0000, 0x00, 0x00, 0x00, 0x00, 0x00})]
-        [InlineData(true, new byte[] {0b0000_0010, 0x00, 0x00, 0x00, 0x00, 0x00})]
-        [InlineData(false, new byte[] {0b1111_1101, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF})]
-        [InlineData(true, new byte[] {0b1111_1111, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF})]
-        public void IsLocallyAdministered_Test(bool expected,
-                                               byte[] bytes)
-        {
-            // Arrange
-            var macAddress = new MacAddress(bytes);
-
-            // Act
-            var isLocallyAdministered = macAddress.IsLocallyAdministered;
+   /// <summary>Tests for <see cref="MacAddress" /></summary>
+   public class MacAddressTests
```

Comment:
Why so many revisions of this file in this PR? Was this because of the recent rule set stuff?

Reply:
Seems likely. I guess I should have rebased onto master before making the PR.